cmake/configs/default.cmake

```diff
@@ -1,314 +1,508 @@
-hunter_config(intltool VERSION 0.51.0)
-hunter_config(ios_sim VERSION 3.1.1)
-hunter_config(ippicv VERSION 20151201)
-hunter_config(irrXML VERSION 1.2)
-hunter_config(kbproto VERSION 1.0.6)
-hunter_config(libdaemon VERSION 0.14)
-hunter_config(libjson-rpc-cpp VERSION 0.7.0-p0)
-hunter_config(libogg VERSION 1.3.2-cmake3)
-hunter_config(libsodium VERSION 1.0.10)
-hunter_config(libuv VERSION 2.0.0-p0)
-hunter_config(libxml2 VERSION 2.9.4)
-hunter_config(libyuv VERSION 1514-p3)
-hunter_config(log4cplus VERSION 1.2.0-p0)
-hunter_config(lzma VERSION 5.2.3-p4)
-hunter_config(mini_chromium VERSION 0.0.1-p2)
-hunter_config(minizip VERSION 1.0.1-p1)
-hunter_config(mpark_variant VERSION 1.0.0)
-hunter_config(msgpack VERSION 1.4.1-p2)
-hunter_config(mtplz VERSION 0.1-p3)
-hunter_config(nanoflann VERSION 1.2.3-p0)
-hunter_config(nlohmann_json VERSION 2.1.1-p1)
-hunter_config(odb VERSION 2.4.0)
-hunter_config(odb-boost VERSION 2.4.0)
-hunter_config(odb-compiler VERSION 2.4.0)
-hunter_config(odb-mysql VERSION 2.4.0)
-hunter_config(odb-pgsql VERSION 2.4.0)
-hunter_config(odb-sqlite VERSION 2.4.0)
-hunter_config(ogles_gpgpu VERSION 0.2.1)
-hunter_config(onmt VERSION 0.4.1-p2)
-hunter_config(openddlparser VERSION 0.1.0-p2)
-hunter_config(pciaccess VERSION 0.13.4)
-hunter_config(poly2tri VERSION 1.0.0)
-hunter_config(polyclipping VERSION 4.8.8-p0) # for Assimp
-hunter_config(presentproto VERSION 1.0)
-hunter_config(pthread-stubs VERSION 0.3)
-hunter_config(pugixml VERSION 1.8.1)
-hunter_config(rabbitmq-c VERSION 0.7.0-p1)
-hunter_config(randrproto VERSION 1.3.2)
-hunter_config(renderproto VERSION 0.11.1)
-hunter_config(sm VERSION 1.2.1)
-hunter_config(Snappy VERSION 1.1.6-p0)
-hunter_config(sse2neon VERSION 1.0.0-p0)
-hunter_config(sparsehash VERSION 2.0.2)
-if(MSVC_VERSION LESS 1800)
-    # for VS12 - version without support C++11
-  hunter_config(spdlog VERSION 1.0.0-p0)
+hunter_default_version(QtAndroidCMake VERSION 1.0.9)
+hunter_default_version(QtCMakeExtra VERSION 1.0.32)
+hunter_default_version(QtPropertyEditor VERSION 2.1.3-p0)
+hunter_default_version(QtQmlManager VERSION 1.0.0)
+hunter_default_version(Qwt VERSION 6.1-p3)
+hunter_default_version(RapidJSON VERSION 1.1.0-66eb606-p0)
+hunter_default_version(RapidXML VERSION 1.13)
+hunter_default_version(SDL2 VERSION 2.0.7-p1)
+hunter_default_version(SDL_image VERSION 2.0.1-p1)
+hunter_default_version(SDL_mixer VERSION 2.0.2-p0)
+hunter_default_version(SDL_ttf VERSION 2.0.14-p0)
+hunter_default_version(SimpleSignal VERSION 0.0.0-79c3f68-p1)
+hunter_default_version(Snappy VERSION 1.1.6-p0)
+hunter_default_version(Sober VERSION 0.1.3)
+hunter_default_version(SuiteSparse VERSION 5.1.2-p0)
+hunter_default_version(TCLAP VERSION 1.2.2-p1)
+hunter_default_version(TIFF VERSION 4.0.2-p5)
+hunter_default_version(Tesseract VERSION 3.05.01-hunter-3)
+hunter_default_version(Urho3D VERSION 1.7-p15)
+hunter_default_version(VulkanMemoryAllocator VERSION 2.1.0-p0)
+hunter_default_version(WDC VERSION 1.1.5)
```

Comment:
This change and in particular https://github.com/ingenue/hunter/pull/327/commits/082a56d93cf10338c2dae51af284032cb940cc1b commit don't belong here, please send it to https://github.com/ruslo/hunter

> I have checked that this pull request contains only
> .travis.yml/appveyor.yml changes. All other changes send
> to https://github.com/ruslo/hunter. [**No**]

It's expected that you will answer **Yes** here.

See also: https://docs.hunter.sh/en/latest/creating-new/update.html#add-toolchains